pytest_bdd/scenario.py

```diff
@@ -292,6 +292,11 @@ def make_python_name(string):
     return re.sub(ALPHA_REGEX, "", string).lower()
 
 
+def make_string(string):
+    """Make a string literal."""
+    return "'{}'".format(string.replace("'", r"\'"))
```

Comment:
From the [python docs about repr](https://docs.python.org/3.7/library/functions.html#repr),

> Return a string containing a printable representation of an object. For many types, this function makes an attempt to return a string that would yield an object with the same value when passed to eval()

So I think that `repr()` is exactly what we should be using here. It will also determine whether to use single or double quotes, according to the string being passed.

Reply:
Yes, it's the better solution. However it will create double-quoted strings, so for consistency I think [scenario.feature.rel_filename](https://github.com/pytest-dev/pytest-bdd/blob/b561f590b553eea49664a48585afdbcc3d32dcc1/pytest_bdd/templates/test.py.mak#L15) should also be double-quoted. (which is something I let black fix after running generate anyway :-) )

What do you think?

Reply:
Yes, definitively.
`repr()` will try to make single-quote strings, and fallback to double-quote strings in case there is already a single quote it the source.
We should not care about `black`ening the resulting files, as they will not be committed in the repository.

Reply:
Ah, I phrased that poorly. What I meant was that when I use `pytest-bdd generate` in my daily work, I usually run black right after.
Anyway, I'll remove the make_string and just use repr inline in the template.

Reply:
`make_string` removed in favor of repr. Then I added make_python_docstring to handle `"""triple"""` quoted strings in docstrings. (overkill? maybe :)) See generate_with_quotes.feature for examples.